hops/featurestore_impl/rest/rest_rpc.py

```diff
@@ -318,6 +318,8 @@ def _update_featuregroup_stats_rest(featuregroup_id, featurestore_id, feature_co
                     constants.REST_CONFIG.JSON_FEATURESTORE_DISABLE_ONLINE_QUERY_PARAM + "=false"
                     + constants.DELIMITERS.AMPERSAND_DELIMITER +
                     constants.REST_CONFIG.JSON_FEATURESTORE_UPDATE_STATISTICS_SETTINGS + "=true"
+                    + constants.DELIMITERS.AMPERSAND_DELIMITER +
+                    constants.REST_CONFIG.JSON_FEATURESTORE_UPDATE_JOB_QUERY_PARAM + "=" + str(len(jobs) > 0)
```

Comment:
Did you test this? I suspect `str(len(jobs) > 0` to return `True`. I am not sure if the upper case T will cause problems in the backend. To be safe you can use `json.dumps(len(jobs) > 0)`

Reply:
Yes, I run both a python job and and a jupyter notebook and it seems to be working - but I'll change it to json just to be safe.